.gitlab-ci.yml

```diff
@@ -5,6 +5,7 @@ before_script:
   - virtualenv ~/venv
   - source ~/venv/bin/activate
   - pip install -r requirements-dev.txt
+  - apt-get install shellcheck
```

Comment:
I almost feel like there should be a `requirements-dev.system` file.

Reply:
That'd be handy!